OlinkAnalyze/R/Olink_color.R

```diff
@@ -2,14 +2,14 @@
 #'
 #' @param alpha transparency (optional)
 #' @param coloroption string, one or more of the following:
-#' c('red', 'orange', 'yellow', 'green', 'teal', 'turqoise', 'gray', 'darkblue', 'purple', 'pink')
```

Comment:
Do we need to change any reference files for the unit tests for this?  I dont think we have a unit test for colors so probably not.

Reply:
We probably should add unit tests for color.

Reply:
I will discuss with @Orbmac  for implementation in a future update and add to Kanban